packages/steps/src/component.tsx

```diff
@@ -9,24 +9,14 @@ export const StepsContext = createContext<{
   horizontal?: boolean;
   right?: boolean;
 }>({
-  horizontal: undefined,
-  right: undefined,
+  horizontal: false,
+  right: false,
 });
 
-export function Steps({ horizontal, right, className, children }: StepsProps) {
```

Comment:
Was it intended to remove className prop from the array in line 19?

Reply:
Oh no! That was not intentional! 😄 🙈  I'll fix it! 😄

Reply:
Updated in this commit: [233d18d](https://github.com/warp-ds/react/pull/274/commits/233d18da5e607b96df723668705f8bfdf160b5a0)